src/legacy/ui/ui_render/ui_render_mixin.js

```diff
@@ -43,12 +43,13 @@ export function uiRenderMixin(kbnServer, server, config) {
   kbnServer.afterPluginsInit(() => {
     const { defaultInjectedVarProviders = [] } = kbnServer.uiExports;
     defaultInjectedVars = defaultInjectedVarProviders
-      .reduce((allDefaults, { fn, pluginSpec }) => (
-        mergeVariables(
+      .reduce((allDefaults, { fn, pluginSpec }) => {
+        const fnValue = fn(kbnServer.server, pluginSpec.readConfigValue(kbnServer.config, []));
+        return mergeVariables(
           allDefaults,
-          fn(kbnServer.server, pluginSpec.readConfigValue(kbnServer.config, []))
```

Comment:
are these changes related to PR? there are a few other places with the same problem.

Reply:
Changes were for debugging, I'll revert.